plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/ViewportPluginImpl.kt

```diff
@@ -37,7 +39,8 @@ import java.util.concurrent.CopyOnWriteArraySet
  *  - transitioning (camera is being managed by a ViewportTransition)
  */
 @MapboxExperimental
-class ViewportPluginImpl : ViewportPlugin {
+class ViewportPluginImpl(private val handler: Handler = Handler(Looper.getMainLooper())) :
```

Comment:
should we add `JvmOverloads`?

Reply:
I don't think this constructor would be used as public API, as the initialisation of plugin instance is done internally in our sdk(but in another module, so need to keep this class public.)